signwriting_animation/diffusion/tests/test_overfitting.py

```diff
@@ -0,0 +1,118 @@
+import torch
+import torch.optim as optim
+import matplotlib.pyplot as plt
+from transformers import CLIPProcessor
+
+from signwriting_animation.diffusion.core.models import SignWritingToPoseDiffusion
+from signwriting_evaluation.metrics.clip import signwriting_to_clip_image
+
+# Set random seed for reproducibility
+torch.manual_seed(42)
+if torch.cuda.is_available():
+    torch.cuda.manual_seed_all(42)
```

Comment:
this is going to repeat in many places, and still not be fully correct.
instead, i recommend 
```py
from lightning import seed_everything

seed_everything(42)
```